README.md

```diff
@@ -1,68 +1,79 @@
+- [Styled Components](https://styled-components.com/)
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
 
-### `yarn test`
+## Installation
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+**1)** Get an Open Weather Map API key
 
-### `yarn build`
+ Sign up to [OpenWeatherMap](https://openweathermap.org/) and get a free key for One Call API.
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
 
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+**2)** Download or fork this repository and install the packages locally with:
+ 
+ ```
+ yarn install
+ ``` 
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+**3)** Setup enviroment variables 
 
-### `yarn eject`
+Create an .env file in the root folder of your project to store both **API key** and **API URL** with these names:
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+ ```
+ REACT_APP_WEATHER_API_KEY = your_api_key_goes_here
+ 
+ REACT_APP_API_URL = https://api.openweathermap.org/data/2.5/onecall
+ ```
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+**4)** Run the app in the development mode. 
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+```
+yarn start
+```
+
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
-## Learn More
+## Usage
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+In order to get a 5-day weather forecast:
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+**1)** **Select** a custom location from the list **OR type** latitude and longitude coordinates in decimal degrees format (99.9999 / -99.9999). (You can find this data for any location here: [latlong.net](https://www.latlong.net/lat-long-dms.html))
 
-### Code Splitting
+**2)** **click Get Weather** 
```

Comment:
Instead of bold, you could use the h3 title from markdown, like `### Title here` (this applies to the other steps too)